src/test/java/SeleniumFramework.java

```diff
@@ -1,51 +1,72 @@
-import org.junit.Test;
+import org.testng.annotations.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
 
-import java.sql.Time;
-import java.util.concurrent.TimeUnit;
 
 public class SeleniumFramework {
 
     @Test
-    public void passwordCheck() {
+    public static void LoginSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
-        driver.get("https://deens-master.now.sh/login");
+        driver.get("https://deens.com/?internaltraffic");
+        WebElement login_button = driver.findElement(By.cssSelector("a[href*='login']"));
+        login_button.click();
+        login_button.sendKeys("Mendygax@gmail.com");
```

Comment:
Please devide login button into login button and e-mail field(use different locators for them). And please don't forget to use camalCase for variables